### ISSUE TITLE
No data propagated from restiming POST request

### ISSUE THREAD

**Issue by philbooth** [nature/boomcatch#61]
Using the test case at:

https://github.com/nature/boomcatch/blob/master/client/restiming-post.html

I'm seeing empty data being passed to the forwarder. The same problem does not exist with `GET` requests.

**Comment by philbooth** [comment]
Body data from the failing request:

```
restiming[0][rt_name]=http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,800,700,600,300&restiming[0][rt_in_type]=link&restiming[0][rt_st]=7.235999975819141&restiming[0][rt_dur]=41.462000051978976&restiming[0][rt_fet_st]=7.235999975819141&restiming[0][rt_res_end]=48.698000027798116&restiming[0][rt_dns_st]=7.235999975819141&restiming[0][rt_dns_end]=7.235999975819141&restiming[0][rt_con_st]=7.235999975819141&restiming[0][rt_con_end]=7.235999975819141&restiming[0][rt_req_st]=9.839000005740672&restiming[0][rt_res_st]=48.21400000946596&restiming[1][rt_name]=http://31.media.tumblr.com/fb876f3b0bd087d90343e2af61f28b8c/tumblr_nlgpkxfK8P1s612eao1_500.gif&restiming[1][rt_in_type]=img&restiming[1][rt_st]=7.313999987673014&restiming[1][rt_dur]=19.315000041387975&restiming[1][rt_fet_st]=7.313999987673014&restiming[1][rt_res_end]=26.62900002906099&restiming[2][rt_name]=http://code.jquery.com/jquery-1.11.3.min.js&restiming[2][rt_in_type]=script&restiming[2][rt_st]=7.382999989204109&restiming[2][rt_dur]=40.55600002175197&restiming[2][rt_fet_st]=7.382999989204109&restiming[2][rt_res_end]=47.93900001095608&u=file:///Users/phil.booth/Code/boomcatch/client/restiming-post.html&v=0.9.1433512298&vis.st=visible
```

**Comment by philbooth** [comment]
The above data, parsed:

```
{ restiming:
   [ { rt_name: 'http://fonts.googleapis.com/css?family=Open Sans:300italic,400italic,600italic,700italic,800italic,400,800,700,600,300',
       rt_in_type: 'link',
       rt_st: '10.700000042561442',
       rt_dur: '30.153999978210777',
       rt_fet_st: '10.700000042561442',
       rt_res_end: '40.85400002077222',
       rt_dns_st: '10.700000042561442',
       rt_dns_end: '10.700000042561442',
       rt_con_st: '10.700000042561442',
       rt_con_end: '10.700000042561442',
       rt_req_st: '13.528000039514154',
       rt_res_st: '40.108000044710934' },
     { rt_name: 'http://31.media.tumblr.com/fb876f3b0bd087d90343e2af61f28b8c/tumblr_nlgpkxfK8P1s612eao1_500.gif',
       rt_in_type: 'img',
       rt_st: '10.772000008728355',
       rt_dur: '17.101000004913658',
       rt_fet_st: '10.772000008728355',
       rt_res_end: '27.873000013642013' },
     { rt_name: 'http://code.jquery.com/jquery-1.11.3.min.js',
       rt_in_type: 'script',
       rt_st: '10.856000008061528',
       rt_dur: '14.714999997522682',
       rt_fet_st: '10.856000008061528',
       rt_res_end: '25.57100000558421' } ],
  u: 'file:///Users/phil.booth/Code/boomcatch/client/restiming-post.html',
  v: '0.9.1433512298',
  vis: { st: 'visible' } }
```

**Comment by philbooth** [comment]
This was caused by using `parseInt` instead of `parseFloat` in `normalise`. Fix will be published as version 3.0.2.
